src/probnum/diffeq/odefiltsmooth/ivpfiltsmooth.py

```diff
@@ -75,13 +76,21 @@ def step(self, t, t_new, current_rv):
 
         return filt_rv, err
 
-    def postprocess(self, times, rvs):
+    def rvlist_to_odesol(self, times, rvs):
```

Comment:
this is now an abstract method in the base class, to enforce that every ODE solver implementation considers creation of ODESolutions.
It partly replaces the previous `postprocess` method (which still exists, but is empty by default).